<?xml version="1.0"?>
<launch>
    <arg name="camera"              value="yumi_l515"/>

    <arg name="camera_frame"        value="$(arg camera)_color_optical_frame"/>

    <arg name="dataset"              value="yumi"/>
    <node pkg="tsl" name="tsl_node" type="tsl_node" respawn="false" output="screen">
        <rosparam command="load" file="$(find tsl)/params/yumi.yaml" if="$(eval arg('dataset') == 'yumi')"/>
    </node>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find tsl)/rviz/tsl_yumi.rviz" />
    <!-- <node pkg="tf" type="static_transform_publisher" name="camera_to_world" args="0 0 0.9 1.5708 0 3.1416 world $(arg camera_frame) 100" /> -->
    <node pkg="tf" type="static_transform_publisher" name="camera_to_world" 
            args="0.11402912172823736, -0.002473922574375894, 0.3408948530896191, 
                -0.6381706638619635, 0.6315954687156851, -0.3108826429470674, 
                0.3117327810907012 yumi_base_link $(arg camera_frame) 100" />

</launch>